### ISSUE TITLE
Error pulling Quen2 models: unknown pre-tokenizer type: 'qwen2'

### ISSUE THREAD

**Issue by agilebean** [ollama/ollama#4969]
### What is the issue?

After downloading any Qwen2 model, e.g.
```
ollama run qwen2:1.5b
```
the following error is shown:

```
Error: llama runner process has terminated: signal: abort trap error:error loading model vocabulary: unknown pre-tokenizer type: 'qwen2'
```
tested on Ollama versions `0.1.38` and `0.1.42`

### OS

macOS

### GPU

Apple

### CPU

Apple

### Ollama version

 0.1.38

**Comment by agilebean** [comment]
This bug could be related to the MacOS app. Removing and reinstalling didn't help.
However, installing with brew did solve the problem.

**Comment by ChenCheng2Cs** [comment]
> This bug could be related to the MacOS app. Removing and reinstalling didn't help. However, installing with brew did solve the problem.

how do you solve it? To be precised, how do you install with brew? Thanks

**Comment by MrSuddenJoy** [comment]
@ChenCheng2Cs `brew install <pkg-name>`. Classic way of installing packages with `brew`.......

**Comment by 0x2CA** [comment]
The same problem,But there's no problem with qwen:14b
```shell
➜ ～ brew search ollama
==> Formulae
ollama

==> Casks
notesollama                ollama ✔                   ollamac
➜  ~ ollama run qwen2:7b
Error: llama runner process has terminated: signal: abort trap error:error loading model vocabulary: unknown pre-tokenizer type: 'qwen2'
```

**Comment by smg1024** [comment]
> The same problem,But there's no problem with qwen:14b
> 
> ```shell
> ➜ ～ brew search ollama
> ==> Formulae
> ollama
> 
> ==> Casks
> notesollama                ollama ✔                   ollamac
> ➜  ~ ollama run qwen2:7b
> Error: llama runner process has terminated: signal: abort trap error:error loading model vocabulary: unknown pre-tokenizer type: 'qwen2'
> ```

@0x2CA 
Reinstall ollama with homebrew `brew reinstall ollama` and restart the service with `brew services restart ollama`
This fixed the problem for me

**Comment by 0x2CA** [comment]
> > The same problem,But there's no problem with qwen:14b
> > ```shell
> > ➜ ～ brew search ollama
> > ==> Formulae
> > ollama
> > 
> > ==> Casks
> > notesollama                ollama ✔                   ollamac
> > ➜  ~ ollama run qwen2:7b
> > Error: llama runner process has terminated: signal: abort trap error:error loading model vocabulary: unknown pre-tokenizer type: 'qwen2'
> > ```
> 
> @0x2CA Reinstall ollama with homebrew `brew reinstall ollama` and restart the service with `brew services restart ollama` This fixed the problem for me

``` shell
➜  0x2ca brew uninstall ollama
==> Uninstalling Cask ollama
==> Backing App 'Ollama.app' up to '/opt/homebrew/Caskroom/ollama/0.1.36/Olla
==> Removing App '/Applications/Ollama.app'
==> Unlinking Binary '/opt/homebrew/bin/ollama'
==> Purging files for version 0.1.36 of Cask ollama
==> Autoremoving 1 unneeded formula:
jsoncpp
Uninstalling /opt/homebrew/Cellar/jsoncpp/1.9.5... (18 files, 300.2KB)
➜  0x2ca brew install ollama
==> Auto-updating Homebrew...
Adjust how often this is run with HOMEBREW_AUTO_UPDATE_SECS or disable with
HOMEBREW_NO_AUTO_UPDATE. Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
==> Auto-updated Homebrew!
Updated 4 taps (homebrew/bundle, homebrew/services, homebrew/core and homebrew/cask).
==> New Formulae
codecov-cli        geni               kubelogin          poutine
==> New Casks
font-beiruti                           font-palemonasmufi-regular
font-palemonasmufi-bold                impel
font-palemonasmufi-bolditalic          xnapper
font-palemonasmufi-italic

You have 3 outdated formulae and 3 outdated casks installed.

Warning: Treating ollama as a formula. For the cask, use homebrew/cask/ollama or specify the `--cask` flag.
==> Downloading https://ghcr.io/v2/homebrew/core/ollama/manifests/0.1.44
###################################################################### 100.0%
==> Fetching ollama
==> Downloading https://ghcr.io/v2/homebrew/core/ollama/blobs/sha256:59b3c5ad
###################################################################### 100.0%
==> Pouring ollama--0.1.44.arm64_sonoma.bottle.tar.gz
==> Caveats
To start ollama now and restart at login:
  brew services start ollama
Or, if you don't want/need a background service you can just run:
  /opt/homebrew/opt/ollama/bin/ollama serve
==> Summary
🍺  /opt/homebrew/Cellar/ollama/0.1.44: 8 files, 22.8MB
==> Running `brew cleanup ollama`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
Removing: /Users/admin/Library/Caches/Homebrew/ollama_bottle_manifest--0.1.43... (7.6KB)
Removing: /Users/admin/Library/Caches/Homebrew/ollama--0.1.43... (8.7MB)
➜  0x2ca brew services restart ollama
Could not enable service: 125: Domain does not support specified action
Error: Failure while executing; `/bin/launchctl enable gui/501/homebrew.mxcl.ollama` exited with 125.
```

**Comment by 0x2CA** [comment]
Oh sorry, I installed it wrong, I reinstalled it and it's working!
`brew install ollama --cask`

**Comment by QiuZiXian** [comment]
升级ollama

**Comment by dhiltgen** [comment]
Support for this model was fixed in a newer version (confirmed on 0.1.44).  Please upgrade and it should work.
